### PR TITLE
Feature/TASK-128 - Improving DbConnetion implementation

### PR DESCRIPTION
**Pull request by cristiang96**
Improved the DbConnection, checking the connection only when it's made for the first time, and raise exceptions in the CRUD operations if needed. @strymsg

### CODE REVIEW

File: task_scheduler/tasks/main.py [hunk 1]
```diff
@@ -65,12 +83,19 @@
...
+# print(db_task.config.db_connection.insert(data3, "api-1"))
+
+# print(db_task.config.db_connection.get("config-*"))
+
+# print(db_task.config.db_connection.delete("config-*"))
+
+# print(db_task.config.db_connection.get("*"))
 
 # redis_client2 = RedisDbConnection(
 #                         db_name="1",
```
Comment (favipcj) [inline task_scheduler/tasks/main.py]: this file is only for testing, right?

Comment (strymsg) [reply task_scheduler/tasks/main.py]: The exception handling is good @cristiang96 , taking favipcj's suggestion the `task_scheduler/tasks/main.py `should be moved to `tests/ `changing its name to something more specific.

Comment (cristiang96) [reply task_scheduler/tasks/main.py]: Yes, that file is only for testing. I moved that file to ```tests\testing_mongo_and_redis_connection.py```.

Comment (strymsg) [reply task_scheduler/tasks/main.py]: Thanks!

**Pull request merged**